### PR TITLE
New button

### PR DESCRIPTION
**Pull request by hemt1912**

### CODE REVIEW

File: html/button.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        height: 40px;
+        width: 100px;
+        background-color: #63d297; 
+        color: #ffffff; 
+        text-align: center; ;
+      }
+      </style>
+   </head>	
+   <body>
+       <div>Summit!</div>
```
Comment (sonhan) [inline html/button.html]: - please use `<button>` element for a button
- please include a screenshot of the button in the PR's description

Comment (hemt1912) [reply html/button.html]: image

Comment (hemt1912) [reply html/button.html]: Done!

File: html/button.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+              width: 100px;
+              background-color: #47e047; 
+              color: #ffffff; 
+              text-align: center">
+            Summit
+          </button>
+   </head>	
+   <body>
+       <div>
+         <MyButton/>
```
Comment (sonhan) [inline html/button.html]: what is this `MyButton`?

File: html/button.html [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+   <head>
+          <button style="
+              border-radius: 25px;
+              height: 40px;
+              width: 100px;
+              background-color: #47e047; 
+              color: #ffffff; 
+              text-align: center">
+            Summit
+          </button>
```
Comment (sonhan) [inline html/button.html]: Move this to `body`

File: html/button.html [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+      <button style="
+      border-radius: 25px;
+      height: 40px;
+      width: 100px;
+      background-color: #47e047; 
+      color: #ffffff; 
+      text-align: center">
+    Summit
+  </button>
+   </body>
```
Comment (sonhan) [inline html/button.html:13]: please fix indentation

**Review by sonhan — APPROVED** [review]

**Pull request merged**